azure-pipelines.yml

```diff
@@ -89,10 +90,12 @@ stages:
     jobs:
       - job: UT_FT_1
         displayName: UT FT common & flink & UT client/spark-client
-        timeoutInMinutes: '120'
+        timeoutInMinutes: '150'
         steps:
           - task: Maven@3
             displayName: maven install
+            continueOnError: true
+            retryCountOnTaskFailure: 1
```

Comment:
I still think that having the `continueOnError` and `retryCount` is useful, otherwise in general people still have to keep triggering azure ci to see the next set of failures, or if theres some azure agent connection issue then have to rerun which also queues up the many builds.

Reply:
Understood.  What you state only applies to your PR which affects most tests.  For other PRs, it's good to fail early on legitimate test errors so that the CI resources can be used to run for other PRs.